docs/running.rst

```diff
@@ -619,32 +619,76 @@ The asynchronous execution policy may provide significant overall performance be
 For compile-only and normal tests that require a compilation, the execution time will be bound by the total compilation time of the test.
 
 
-
-Mapping modules
+Manipulating modules
 --------------------
 
-When invoking ReFrame, user-defined mappings of modules can be passed in order to replace the modules specified in a regression test without the need of modifying the files. This feature enables regression tests to be module-agnostic, which can be very practical in frequent use cases like, for instance, when performing regression tests considering different versions of a module.
+.. versionadded:: 2.11
+
+ReFrame allows you to change the modules loaded by a regression test on-the-fly without having to edit the regression test file.
+This feature is extremely useful when you need to quickly test a newer version of a module, but it also allows you to completely decouple the module names used in your regression tests from the real module names in a system, thus making your test even more portable.
+This is achieved by defining *module mappings*.
+
+There are two ways to pass module mappings to ReFrame.
+The first is to use the ``--map-module`` command-line option, which accepts a module mapping.
+For example, the following line defines maps the module ``test_module`` to the module ``real_module``:
```

Comment:
'the following line defines maps the module'.

Maybe 'the following line defines a mapping of the module ... to the module ...'?

Reply:
I would just remove the `defines`.